flink-table/flink-table-planner/src/main/java/org/apache/calcite/sql2rel/SqlToRelConverter.java

```diff
@@ -6088,8 +6088,11 @@ private void translateAgg(
                 // switch out of agg mode
                 bb.agg = null;
                 // ----- FLINK MODIFICATION BEGIN -----
-                List<SqlNode> sqlNodes =
-                        new SqlCallBinding(validator(), aggregatingSelectScope, call).operands();
+                SqlCallBinding sqlCallBinding =
+                        new SqlCallBinding(validator(), aggregatingSelectScope, call);
+                List<SqlNode> sqlNodes = sqlCallBinding.operands();
```

Comment:
I think we can just override the operands() here and return a special SqlNode(whose return type is not ANY)if its operator is DEFAULT.